implicit.cabal

```diff
@@ -43,6 +43,7 @@ Library
         -rtsopts
         -funfolding-use-threshold=16 
         -fspec-constr-count=10
+        -fno-warn-tabs
```

Comment:
On Sun, Jun 28, 2015 at 05:19:52PM -0700, Jonathan Merritt wrote:
> Drive-by review: Has `-fno-warn-tabs` been added because there are tabs in the current code?

Yep.

> It might be useful to remove them as well, as part of a 7.10 patch.

I don't see tabs as a bad thing anyway.

-- 
()  ascii ribbon campaign - against html mail
/\  http://arc.pasp.de/   - against proprietary attachments